### PR TITLE
Add missing documentation dependencies

### PR DESCRIPTION
**Pull request by xoriole**
~~This is a sample PR to test the fix for https://github.com/Tribler/tribler/issues/6095.~~

It includes some missing dependencies which should be merged for successful building of the docs.

@devos50 thank you Martijn for pointing out the missing mock modules.

### CODE REVIEW

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

File: doc/conf.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def __getattr__(cls, name):
                 'meliae', 'libnacl', 'decorator', 'libnacl.dual', 'libnacl.sign', 'libnacl.encode', 'libnacl.public',
                 'netifaces', 'ipv8.messaging.anonymization.tunnel', 'Tribler.community.gigachannel.community',
                 'networkx', 'validate', 'ipv8.REST.root_endpoint', 'anydex.restapi.root_endpoint',
-                'anydex.restapi.wallets_endpoint']
+                'anydex.restapi.wallets_endpoint', 'ipv8_service', 'faker', 'sentry_sdk',
```
Comment (devos50) [inline doc/conf.py:58]: You can remove the `anydex.restapi.wallets_endpoint` mock (Anydex is not included in Tribler anymore) 👍

**Review by devos50 — CHANGES_REQUESTED** [review]

File: doc/conf.py (hunk 1 above)
Comment (xoriole) [reply doc/conf.py]: Good point 👍 
Removed

**Review by devos50 — APPROVED** [review]

**Pull request merged**